### DISCUSSION TITLE
Rsync speed

### DISCUSSION THREAD

**Discussion by vvavepacket** [ericaltendorf/plotman#420] [Q&A]
Hows the archival / rsync speed? Is it fast? Or limited/ not using the full bandwidth?

I heard rsync is slow even with no compression, its not saturating the whole 1gbit link and you are getting 40MB/SEC max as opposed to just network drive/samba which can reach full speed.

Anyone tried and measured the speed?

**Comment by altendky** [comment]
Depending on the day I can get 70MB/s over WiFi.  But yeah, we are working on allowing arbitrary commands to be configured as the archival transfer command.

**Reply by lnb2g** [reply]
really? mine is capped at 1MB/s when on 10MB/s port and about 12MB/s when on 100MB/s port

**Comment by thomann061** [comment]
Over a wired 1Gigabit ethernet I can reach up to 125MB/s. The bottleneck I currently experience is when my buffer drive (hdd) is being written to and writing to the archive at the same time. At least this is what I think is bottlenecking it. I am going to swap out the hdd with an ssd and see if that improves.

**Reply by lnb2g** [reply]
internal or external? External depending on your usb or sata connection as well.
